clocker.bom

```diff
@@ -49,6 +49,18 @@ brooklyn.catalog:
           $brooklyn:entitySpec:
             type: docker-host
 
+  - id: docker-infrastructure-with-registry
```

Comment:
Should have a `docker-registry` catalog entry` and then add the config to a new template for a `docker-cloud-with-registry` as an example.